tests/certification/bindings/azure/eventhubs/eventhubs_test.go

```diff
@@ -369,7 +379,7 @@ func TestEventhubBindingMultiplePartition(t *testing.T) {
 			outputmsg2[i] = fmt.Sprintf("output binding: Message %03d", i)
 		}
 		consumerGroup2.ExpectStrings(outputmsg2...)
-		time.Sleep(40 * time.Second)
+		time.Sleep(120 * time.Second)
```

Comment:
As discussed offline, in this test, please send 50 events to partition 0 (via the component default) and 50 events to partition 1 via the event metadata partitionkey override. Then verify only the desired events went to the appropriate partition and nowhere else.